ksqldb-cli/src/main/resources/log4j2.yaml

```
@@ -0,0 +1,16 @@
+Configuration:
+  status: WARN
+  Appenders:
+    File:
```

```suggestion
    RollingFile:
```
old one is io.confluent.ksql.util.TimestampLogFileAppender

---

We will use TimestampLogFileAppender